server/tests/tasks/test_repo_metadata_bitbucket.py

```
@@ -0,0 +1,63 @@
+    mock_cls.return_value = _client({"error": True, "status": 403})
+    assert _fetch_bitbucket_context("tok", "api_token", "ws", "priv") == (
+        "",
+        "(could not list files)",
+    )
+
+
+@patch("utils.repo_metadata._generate_summary")
+@patch("utils.repo_metadata._update_metadata")
+@patch("utils.repo_metadata._fetch_repo_context", return_value=("", ""))
+@patch("utils.repo_metadata._get_credentials", return_value={"access_token": "x"})
+def test_empty_repo_marked_ready(get_creds, fetch, update, summarize):
```

Ruff (0.16.1)

[warning] Unused function argument: `get_creds`

(ARG001)